bears/general/InvalidLinkBear.py

```diff
@@ -33,14 +44,15 @@ def find_links_in_file(file):
             match = regex.search(line)
             if match:
                 link = match.group()
-                code = InvalidLinkBear.get_status_code_or_error(link)
+                code = self.get_status_code_checked(link)
                 yield line_number + 1, link, code
 
     def run(self, filename, file):
         '''
         Yields results for all invalid links in a file.
         '''
-        for line_number, link, code in InvalidLinkBear.find_links_in_file(file):
+        for line_number, link, code in (InvalidLinkBear.find_links_in_file(
+                                        self, file)):
```

Comment:
so you had to change the implementation of `find_links_in_file` right? that does correspond to how it is called from within other functions, so i don't see how changing `InvalidLinkBear.find_links_in_file(file)` , to `InvalidLinkBear.find_links_in_file(self, file)`  makes sense in this commit and why changing it to self.find_links_in_file(file) deserves a separate commit?